app/uk/gov/hmrc/debttransformationstub/controllers/DebtManagementAPITestController.scala

```diff
@@ -57,4 +54,37 @@ class DebtManagementAPITestController @Inject() (
         }
   }
 
+  def getDebtDataAndDWISignals(wmfId: String): Action[AnyContent] = Action.async { request =>
+    if (appConfig.isPollingEnv)
+      pollingService.insertRequestAndServeResponse(Json.obj(), request.uri).map {
```

Comment:
This is inserting an empty object, is that correct?

Reply:
NVM looks like it's on purpose, I think I understand

Reply:
Yes it's only a get request so will always be empty body